### PR TITLE
Create Digest Article “garys-digest-article”

### PR DESCRIPTION
**Pull request by mirhamasala**
Automatically generated by Decap CMS

### CODE REVIEW

File: src/content/digest/garys-digest-article.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  - first-name: Gary
+    last-name: Moran
+    company: Gary Co.
+seo:
+  twitter:
+    card: summary
+  description: Description
+---
+
+Content
```
Comment (mirhamasala) [inline src/content/digest/garys-digest-article.md:18]: I think we could elaborate a bit here.

File: src/content/digest/garys-digest-article.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+title: Gary's Digest Article
```
Comment (mirhamasala) [inline src/content/digest/garys-digest-article.md:2]:
```suggestion
title: Gary Moran's Digest Article
```

File: src/content/digest/garys-digest-article.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+created-on: 2024-12-06T14:21:00.000Z
+updated-on: 2024-12-06T14:21:00.000Z
+published-on: 2024-12-06T14:21:00.000Z
+issue-number: "1"
+article-number: 1
+authors:
+  - first-name: Gary
+    last-name: Moran
+    company: Gary Co.
```
Comment (gmoranxyz) [inline src/content/digest/garys-digest-article.md:11]:
```suggestion
    company: Gary Inc.
```

^^ this is the correct name for my company

File: src/content/digest/garys-digest-article.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+title: Gary Moran's Digest Article
```
Comment (mirhamasala) [inline src/content/digest/garys-digest-article.md:2]:
```suggestion
title: Gary Moran's Digest Article!
```

File: src/content/digest/garys-digest-article.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+created-on: 2024-12-06T14:21:00.000Z
+updated-on: 2024-12-06T14:21:00.000Z
+published-on: 2024-12-06T14:21:00.000Z
+issue-number: "1"
+article-number: 1
+authors:
+  - first-name: Gary
```
Comment (mirhamasala) [inline src/content/digest/garys-digest-article.md:9]:
```suggestion
  - first-name: Gary Famous For A Day
```

**Pull request closed without merging**